platform.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2014-present PlatformIO <contact@platformio.org>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import subprocess
+
+from platformio import proc
+from platformio.managers.platform import PlatformBase
+
+def is_pyocd_installed():
+    """Check if pyOCD is installed."""
+    try:
+        import pyocd
+        return pyocd.__version__ == "0.36.0"
+    except ImportError:
+        return False
+
+def install_pyocd():
+    """Install pyOCD package."""
+    args = [
+        proc.get_pythonexe_path(),
+        "-m",
+        "pip",
+        "install",
+        "pyocd==0.36.0"
+    ]
+
+    return subprocess.call(args) == 0
+
+class Hc32f46xPlatform(PlatformBase):
+
+    def configure_default_packages(self, variables, targets):
+        # install pyOCD 0.36.0 package
+        # TODO this is a workaround needed until PlatformIO decided they'd want to support 
```

Comment:
can this be requested?

Reply:
created thread in platformio forums https://community.platformio.org/t/update-tool-pyocd-package/39857

Reply:
platformio actually just ... updated the tool package?! 

Well, time to remove the hacky business and use the official package then...